### DISCUSSION TITLE
Fetching a <ul> HTML element under an anchor link

### DISCUSSION THREAD

**Discussion by zkbgt** [FreshRSS/FreshRSS#6948] [Q&A]
Using the "Article CSS selector on original website" to fetch article content, can I fetch content from a single **ul** element under the relevant anchor link when the page is a continuous feed with all anchor links for previous posts?
The example feed I am trying this on is here: https://ctrl-c.club/~pgadey/soc/
So for example for the [2024-10-17 post](https://ctrl-c.club/~pgadey/soc/#2024-10-17) I would expect to see:

> [[11:24]](https://ctrl-c.club/~pgadey/soc/index.html#2024-10-17-11:24) I've written [elsewhere](https://pgadey.ca/blog/gem/) about gratitude, exercise, and meditation. However, I don't think I've appreciated how small the dosage can be. Today, I was feeling totally crummy and asked myself: "How could I feel in fifteen minute?". I then went ahead and did five minutes of each thing and felt a LOT better. Fifteen minutes isn't a tonne of time. It might be worth trying out sometime, if you're feeling crummy.
> [[11:22]](https://ctrl-c.club/~pgadey/soc/index.html#2024-10-17-11:22) Reddit: [What knots do you use while camping?](https://old.reddit.com/r/knots/comments/1g4av79/what_knots_do_you_use_while_camping/)

I have put in "ul" as my CSS selector, however that pulls in posts from all days.
Hoping I've explained that OK. Any help appreciated.

**Comment by Alkarex** [comment]
The anchor element in `https://ctrl-c.club/~pgadey/soc/#2024-10-17` simply indicates a position in a single HTML page, and when it comes to retrieving content, it is the same as `https://ctrl-c.club/~pgadey/soc/`
I cannot think of a way to make it purely with a CSS selector.
Our HTML+XPath mode could potentially work. However, the page `https://ctrl-c.club/~pgadey/soc/` has also invalid HTML (as visible by the wrong hierarchy of dates), so it complicates a bit the matter.

**Reply by zkbgt** [reply]
Thanks for the quick response Alexandre. 
I will have a look at HTML-XPath method and report back if I find a solution. However, sounds like it might be a lost cause. 
Appreciate you looking into it.

**Reply by Frenzie** [reply]
It isn't quite clear to me what you want and presumably the CSS parser doesn't currently support the `:has()` pseudoselector, but if it does you can probably do what it is you're looking for.
